example/Demo.vue

```diff
@@ -193,7 +193,13 @@
 
     <div class="example">
       <h3>Inline datepicker</h3>
-      <datepicker :inline="true"></datepicker>
+      <datepicker :inline="true"
```

Comment:
Could you add this to a new example demo block please?

Reply:
Will do